src/schema/rules/sidecars.yml

```diff
@@ -0,0 +1,103 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+
+---
+MRIScannerHardware:
+    selectors:
+        - modality == "MRI"
+    fields:
+        Manufacturer:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 0070 `Manufacturer`.
+        ManufacturersModelName:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 1090 `Manufacturers Model Name`.
+        DeviceSerialNumber:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0018, 1000 `DeviceSerialNumber`.
+        StationName:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 1010 `Station Name`.
+        SoftwareVersions:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0018, 1020 `Software Versions`.
+        HardcopyDeviceSoftwareVersion: DEPRECATED
+        MagneticFieldStrength:
+            level: RECOMMENDED, but REQUIRED for Arterial Spin Labeling
+        ReceiveCoilName: RECOMMENDED
+        ReceiveCoilActiveElements: RECOMMENDED
+        GradientSetType: RECOMMENDED
+        MRTransmitCoilSequence: RECOMMENDED
+        MatrixCoilMode: RECOMMENDED
+        CoilCombinationMethod: RECOMMENDED
+
+MRISequenceSpecifics:
+    selectors:
+        - modality == "MRI"
+    fields:
+        PulseSequenceType: RECOMMENDED
+        ScanningSequence: RECOMMENDED
+        SequenceVariant: RECOMMENDED
+        ScanOptions: RECOMMENDED
+        SequenceName: RECOMMENDED
+        PulseSequenceDetails: RECOMMENDED
+        NonlinearGradientCorrection: RECOMMENDED, but REQUIRED if [PET](./09-positron-emission-tomography.md) data are present
+        MRAcquisitionType: RECOMMENDED, but REQUIRED for Arterial Spin Labeling
+        MTState: RECOMMENDED
+        MTOffsetFrequency: OPTIONAL
+        MTPulseBandwidth: OPTIONAL
+        MTNumberOfPulses: OPTIONAL
+        MTPulseShape: OPTIONAL
+        MTPulseDuration: OPTIONAL
+        SpoilingState: RECOMMENDED
+        SpoilingType: OPTIONAL
+        SpoilingRFPhaseIncrement: OPTIONAL
+        SpoilingGradientMoment: OPTIONAL
+        SpoilingGradientDuration: OPTIONAL
+
+PETMRISequenceSpecifics:
+    selectors:
+        - modality == "MRI"
+        - "PET" in dataset.modalities
```

Comment:
Maybe this would be more general for selectors?

```yaml
selectors:
  - field: modality
    equals: "MRI"
  - field: "dataset.modalities"
    in: "PET"
```

Reply:
Here's another example from Kubernetes yaml config parser with some interesting structure for this.

```yaml
selector:
  - {key: modality, operator: Eq, values: "MRI"}
  - {key: "dataset.modalities", operator: In, values: "PET"}
```

Reply:
I'm inclined to stick with a simple language if possible; as we work through all the tables, we'll see how complex they get.

Reply:
I agree w/ @effigies for now that the `selectors` are written in something straight forward that that will say work w/ `eval( <selector 1 >)` in JS. It clearly (to this reader) captures the desired state and can always be updated to something more parse-able like the examples @nellh provided.